### PR TITLE
typos fixed - https://github.com/vlajos/misspell-fixer

### PR DESCRIPTION
**Pull request by vlajos**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

- [X] Documentation content changes

## What is the current behavior?

Fix some typos.

## What is the new behavior?

No change is expected.

## Does this PR introduce a breaking change?

- [x] No

### CODE REVIEW

File: packages/compiler/src/expression_parser/parser.ts [hunk 1]
```diff
@@ -670,7 +670,7 @@ export class _ParseAST {
   }
 
   /**
-   * An identifier, a keyword, a string with an optional `-` inbetween.
+   * An identifier, a keyword, a string with an optional `-` between.
```
Comment (gkalpak) [inline packages/compiler/src/expression_parser/parser.ts]: "Between" sounds weird here. Maybe drop it altogether or change it to `in between`.

File: packages/core/src/util/decorators.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ export interface TypeDecorator {
 }
 
 export const ANNOTATIONS = '__annotations__';
-export const PARAMETERS = '__paramaters__';
+export const PARAMETERS = '__parameters__';
```
Comment (gkalpak) [inline packages/core/src/util/decorators.ts]: I wonder if this change would break anything 🤔

File: aio/src/assets/js/prettify.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ S=/^(DIR|FILE|array|vector|(de|priority_)?queue|(forward_)?list|stack|(const_)?(
...
-hashComments:!0,multiLineStrings:!0,regexLiterals:2}),["perl","pl","pm"]);t(y({keywords:R,hashComments:!0,multiLineStrings:!0,regexLiterals:!0}),["rb","ruby"]);t(y({keywords:F,cStyleComments:!0,regexLiterals:!0}),["javascript","js","ts","typescript"]);t(y({keywords:"all,and,by,catch,class,else,extends,false,finally,for,if,in,is,isnt,loop,new,no,not,null,of,off,on,or,return,super,then,throw,true,try,unless,until,when,while,yes",hashComments:3,cStyleComments:!0,multilineStrings:!0,tripleQuotedStrings:!0,
+hashComments:!0,multiLineStrings:!0,regexLiterals:2}),["perl","pl","pm"]);t(y({keywords:R,hashComments:!0,multiLineStrings:!0,regexLiterals:!0}),["rb","ruby"]);t(y({keywords:F,cStyleComments:!0,regexLiterals:!0}),["javascript","js","ts","typescript"]);t(y({keywords:"all,and,by,catch,class,else,extends,false,finally,for,if,in,is,isn't,loop,new,no,not,null,of,off,on,or,return,super,then,throw,true,try,unless,until,when,while,yes",hashComments:3,cStyleComments:!0,multilineStrings:!0,tripleQuotedStrings:!0,
```
Comment (gkalpak) [inline aio/src/assets/js/prettify.js]: This change is invalid.

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: packages/core/src/util/decorators.ts (hunk 2 above)
Comment (vlajos) [reply packages/core/src/util/decorators.ts]: I am not too sure to be honest. Should I revert it?

File: packages/compiler/src/expression_parser/parser.ts (hunk 1 above)
Comment (vlajos) [reply packages/compiler/src/expression_parser/parser.ts]: I have modified it.

File: aio/src/assets/js/prettify.js (hunk 3 above)
Comment (vlajos) [reply aio/src/assets/js/prettify.js]: Oops. Thanks. I have reverted it.

File: packages/core/src/util/decorators.ts (hunk 2 above)
Comment (gkalpak) [reply packages/core/src/util/decorators.ts]: It probably won't break anything. Let's wait for one `core` and `compiler` owners to confirm.

**Review by gkalpak — APPROVED** [review]

**Comment by ocombe** [comment]
:o impressive! so many typos!

**Comment by gkalpak** [comment]
And imagine this is the second pass (here is the first: #16257) :smiley:

**Review by IgorMinar — APPROVED** [review]
Thank you so much for this!

File: packages/core/src/util/decorators.ts (hunk 2 above)
Comment (IgorMinar) [reply packages/core/src/util/decorators.ts]: I think it's fine.

**Comment by IgorMinar** [comment]
Caretaker note: can you please run a g3 presubmit on this?

**Pull request closed without merging**